.github/workflows/ci.yml

```
@@ -391,6 +393,54 @@ jobs:
           name: webapp-client
           path: webapp/client/
 
+
+  devolutions-gateway-players:
+    name: devolutions-gateway-players
```

suggestion: Typically, we set `name` to a more "friendly name".
```suggestion
    name: Recording Player
```